src/components/API.js

```diff
@@ -0,0 +1,172 @@
+// b73a4638-eb7a-419c-8b34-f34f4d770b05 auth token
+
+export default class Api {
+  constructor(key, url) {
+    this._key = key;
+    this._url = url;
+  }
+
+  getPromiseAll(funcs) {
+    return Promise.all(funcs)
+  }
+
+  // Profile
+
+  getUserInfo() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/users/me", {
+      headers: {
+        authorization: this._key
+      }
+    })
+    .then(res => {
+      if (res.ok) {
+        return res.json();
+      }
+      return Promise.reject(`Error: ${res.status}`);
+    })
+    .catch((err) => {
```

Comment:
You should not use `catch` in `Api` because you need to `catch` possible errors at the very **end** of any request, and this end is in `index.js` (`App.js`) after all `then` blocks. And in real projects blocks `catch` usually open error popups and so on.